src/main/resources/application.conf

```diff
@@ -32,27 +18,36 @@ spray.can {
   }
 }
 
+workflow-options {
+  // These workflow options will be encrypted when stored in the database
+  encrypted-fields: ["refresh_token"]
+
+  // AES-256 key to use to encrypt the values in `encrypted-fields`
+  base64-encryption-key: "q78yJ9CtbWTX3+zDHJgHSiN4I7+nq5kgYjOD94nWK48="
```

Comment:
Well for one thing it's accessible in a public github :) It's also in the jar.

The idea is that application.conf should contain a bog standard public default for joe public. Anything that's specific to our internal setup and/or private/secret should exist in the other conf which overrides application.conf when the server launches